### DISCUSSION TITLE
Add a way to customize how a value's property is displayed

### DISCUSSION THREAD

**Discussion by michael-wolfenden** [datalust/seq-tickets#1110] [Ideas]
So our logs typically consist of a number of identification properties (for example `CustomerId`, `ActivityId`).

Right now we use userscripts / greasemonkey / tampermonkey scripts to replace these ids with links to their representations in our system

For example `EmailActivityId` below is now a link to the email activity in our internal system

![image](https://cloud.githubusercontent.com/assets/780521/25974183/feda9fd6-36ea-11e7-9e0d-d03a4aad7fba.png)

It would be nice to somehow have the ability to configure this in seq.

I looked at doing this by using a Seq App, however I don't believe you can add a new property to an existing event (with this link) and even if you could, I would not really want these extra properties persisted.

This feature could have other uses as well.

Currently we log elapsed times in milliseconds, but that makes it hard to read in the UI

![image](https://cloud.githubusercontent.com/assets/780521/25974324/d57ed354-36eb-11e7-920d-a744f11b7c50.png)

This feature could potentially be used to format it in a human readable format like `2 minutes`.

**Comment by nblumhardt** [comment]
Thanks for the suggestion 👍 

This is similar to #199 - but I think this ticket emphasizes the user-customizability of the feature that's not mentioned in the earlier ticket.

**Comment by nblumhardt** [comment]
I'm still a fan of this one but sadly, it's not able to be pulled into a release "standalone", and so we'll drop it from the backlog until a larger chunk of UI rework comes our way. Thanks @michael-wolfenden !

**Comment by nblumhardt** [comment]
Related: https://docs.datalust.co/discuss/5edee692e4a3b00039125d55

This would be a good candidate to consider in our next round of UI updates.

**Comment by andymac4182** [comment]
This would be great with some of the Tracing properties as you could potentially enhance Seq to either link to an external tracing platform or build up a seq query for what you want. :)

**Comment by nblumhardt** [comment]
CC @KodrAus RE tracing primitives ... :-)

**Comment by a98c14** [comment]
Is there any updates on this?
